content/en/apps/guides/hosting/monitoring/setup.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Grafana and Prometheus Setup"
+linkTitle: "Setup"
+description: >
+    Setting up Grafana and Prometheus with the CHT
+---
+
+{{% pageinfo %}} 
+These instructions apply to both CHT 3.x (beyond 3.12) and CHT 4.x.  
+{{% /pageinfo %}}
+
+## Grafana and Prometheus
+
+Medic maintains an opinionated configuration of [Prometheus](https://prometheus.io/) (including [json_exporter](https://github.com/prometheus-community/json_exporter)) and [Grafana](https://grafana.com/grafana/) which can easily be deployed using Docker. It is supported on CHT 3.12 and later, including CHT 4.x.  By using this solution a CHT deployment can easily get longitudinal monitoring and push alerts using Email, Slack or other mechanisms.  All tools are open source and have no licensing fees.
+
+The solution provides both an overview dashboard as well as a detail dashboard.  Here is a portion of the overview dashboard:
+
+![Screenshot of Grafana Dashboard showing data from Prometheus](monitoring.and.alerting.screenshot.png)
+
+[Prometheus supports](https://prometheus.io/docs/concepts/metric_types/) four metric types: Counter, Gauge, Histogram, and Summary.  Currently, the CHT only provides Counter and Gauge type metrics. When building panels for Grafana dashboards, [Prometheus Functions](https://prometheus.io/docs/prometheus/latest/querying/functions/) can be used to manipulate the metric data. Refer to the [Grafana Documentation](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/best-practices/) for best practices on building dashboards.
+
+
+### Prerequisites
+
+- [Docker](https://docs.docker.com/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- URL(s) of the CHT instance(s)
+
+### Setup
+
+These instructions have been tested against Ubuntu, but should work against any OS that meets the prerequisites. They follow a happy path assuming you need to only set a secure password and specify the URL(s) to monitor:
+
+1. Run the following commands to clone this repository, initialize your `.env` file, create a secure password and create your data directories:
+
+    ```sh
+    cd ~
+    git clone https://github.com/medic/cht-monitoring.git
+    cd cht-monitoring
+    cp cht-instances.example.yml cht-instances.yml
+    cp grafana/grafana.example.ini grafana/grafana.ini
+    mkdir -p grafana/data && mkdir  -p prometheus/data 
+    apt install -y wamerican  # ensures /usr/share/dict/words is present for shuf call below 
```

Comment:
Is it just me that still has to `sudo` apt commands?

Reply:
fair! 3933f29e